### ISSUE TITLE
Config not found (local installation)

### ISSUE THREAD

**Issue by DennisKo** [TheLastBen/fast-stable-diffusion#269]
I am trying to get a local installation working but I am stuck when training:

```
File "/opt/conda/lib/python3.7/site-packages/transformers/configuration_utils.py", line 604, in _get_config_dict
    user_agent=user_agent,
  File "/opt/conda/lib/python3.7/site-packages/transformers/utils/hub.py", line 297, in cached_path
    raise EnvironmentError(f"file {url_or_filename} not found")
OSError: file /content/stable-diffusion-v1-5/config.json not found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/content/diffusers/examples/dreambooth/train_dreambooth.py", line 694, in <module>
    main()
  File "/content/diffusers/examples/dreambooth/train_dreambooth.py", line 442, in main
    text_encoder = CLIPTextModel.from_pretrained(args.pretrained_model_name_or_path, subfolder="text_encoder")
  File "/opt/conda/lib/python3.7/site-packages/transformers/modeling_utils.py", line 1595, in from_pretrained
    **kwargs,
  File "/opt/conda/lib/python3.7/site-packages/transformers/models/clip/configuration_clip.py", line 126, in from_pretrained
    config_dict, kwargs = cls.get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/transformers/configuration_utils.py", line 548, in get_config_dict
    config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/transformers/configuration_utils.py", line 637, in _get_config_dict
    f"Can't load config for '{pretrained_model_name_or_path}'. If you were trying to load it from "
OSError: Can't load config for '/content/stable-diffusion-v1-5'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure '/content/stable-diffusion-v1-5' is the correct path to a directory containing a config.json file
```

I am pretty sure I have the correct folder setup. I double-checked with colab settings...

Where is that `config.json` supposed to be? I cant find it in a colab environment either...

**Comment by TheLastBen** [comment]
it's looking for the model in : /content/stable-diffusion-v1-5
change the MODEL_NAME to the folder where you downloaded the model

**Comment by DennisKo** [comment]
> it's looking for the model in : /content/stable-diffusion-v1-5
change the MODEL_NAME to the folder where you downloaded the model

That's the thing. the model is exactly in that folder

```
base:/content/stable-diffusion-v1-5$ ls -a
.  ..  feature_extractor  model_index.json  safety_checker  scheduler  text_encoder  tokenizer  unet  vae
```

**Comment by TheLastBen** [comment]
Check inside the subfolders if the contain all the files needed like the *.json files and the *.bin files

**Comment by DennisKo** [comment]
I think its all there. Again checked with a colab env
```
.
├── feature_extractor
│   └── preprocessor_config.json
├── model_index.json
├── safety_checker
│   ├── config.json
│   └── pytorch_model.bin
├── scheduler
│   └── scheduler_config.json
├── text_encoder
│   ├── config.json
│   └── pytorch_model.bin
├── tokenizer
│   ├── merges.txt
│   ├── special_tokens_map.json
│   ├── tokenizer_config.json
│   └── vocab.json
├── unet
│   ├── config.json
│   └── diffusion_pytorch_model.bin
└── vae
    ├── README.md
    ├── config.json
    └── diffusion_pytorch_model.bin
```

**Comment by DennisKo** [comment]
I think its something in the transformers package: here `transformers/tokenization_utils_base.py` one of the dicts has an item `('added_tokens_file', None)`.
Which version of transformers is running on the colab env?

**Comment by DennisKo** [comment]
Ok fixed by running `pip install "transformers==4.21.0" --no-deps`

**Comment by ZeroCool22** [comment]
> pip install "transformers==4.21.0" --no-deps

That command should be run here?:

`(base) C:\Users\ZeroCool22\Desktop\Ben\fast-stable-diffusion>pip install "transformers==4.21.0" --no-deps`
